ZukoAzula2016/src/org/team1540/zukoazula/HeadingSensor.java

```diff
@@ -0,0 +1,20 @@
+package org.team1540.zukoazula;
+
+import org.team1540.drivers.kauailabs.FRCAHRS;
+
+import ccre.channel.FloatInput;
+import ccre.cluck.Cluck;
+
+public class HeadingSensor {
+    private static final FRCAHRS sensor = FRCAHRS.newSPI_MXP();
+
+    public static final FloatInput yawAngle = sensor.getYawAngle();
+    public static final FloatInput yawRate = sensor.getYawRate();
+
+    public static void setup() {
+        Cluck.publish("Yaw Angle", yawAngle);
```

Comment:
Can you prefix everything with "Heading" so that if we add more it is all grouped together?